common/extensions/api/brave_rewards.json

```diff
@@ -658,6 +658,10 @@
                       "type": "number",
                       "minimum": 0
                     }
+                  },
+                  "payoutStatus": {
+                    "type": "any",
```

Comment:
This should probably be "object".

Reply:
I can't find documentation for this JSON schema, but I think there's a way. I did find [this](https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/extensions/api/action.json;l=96). Maybe something like this would work?

```
"type": "object",
"additionalProperties": { "type": "string" }
```

Reply:
That looks perfect, thank you!